docs/articles/documentation/recipes/integrating-testcafe-with-ci-systems/azure-devops.md

```diff
@@ -0,0 +1,203 @@
+---
+layout: docs
+title: Integrating TestCafe with Azure DevOps
+permalink: /documentation/recipes/integrating-testcafe-with-ci-systems/azure-devops.html
+---
+# Integrating TestCafe with Azure DevOps
+
+This topic describes how to integrate TestCafe tests into an [Azure DevOps Server](https://azure.microsoft.com/en-us/services/devops/server/) and [Azure Pipelines](https://azure.microsoft.com/en-us/services/devops/pipelines/) project's build process.
```

Comment:
Should we add some note for those guys who asked us about TFS integration (why we provide this recipe instead)?

Reply:
Well, I begin thinking that we need a recipe for TFS as well.

Reply:
:-)

Probably, we should describe why we don't have a recipe for TFS (just to add search keywords for those who look for such a recipe).

Reply:
Even if we never write about TFS, how is this explanation going to help people who use TFS?